This is a comment blog on the newly published paper [Marsden et al. 2016](http://www.pnas.org/content/113/1/152.short).

**Title: Bottlenecks and selective sweeps during domestication have increased deleterious genetic variation in dogs**

This is a paper on population genomics of dogs. It described the pattern of lower neutral diversity,  higher diversity at non-synonymous sites. By digging into the cause of the pattern, the authors excludes the role of inbreeding. On one hand,  removing the large Runs of Homozygosity (> 2MB, indicating the recent inbreeding events)  did not change the pattern. On the other hand, forward simulation with high inbreeding coefficients and no bottleneck effect could not produce the similar pattern. And the author concluded several bottlenecks in dogs is the main cause of the pattern.

The author then quantified the genetic burden in dogs through [GERP](http://mendel.stanford.edu/SidowLab/downloads/gerp/) scores. But they only counted the non-synonymous sites with GERP>0. The wolfs have more deleterious amino acid changing  heterozygous sites than dogs, but dogs have more deleterious amino acid changing  homozygous sites than wolfs. Still, dogs have more deleterious alleles than wolfs (heterozygous sites as once, and homozygous sites as twice). If they counted the synonymous sites with GERP>0, they did not find difference between dogs and wolfs. So, the authors deducted it should not be solely owing to the elevated drift in dog populations, as the increased genetic load should also reflect on the synonymous deleterious sites.

The last interesting point of the paper is the enriched non-synonymous deleterious sites and mendelian disease in the sweep regions than the non-sweep regions, which draws the attention on the dog breeding. I like the part how the population genetic research could be practical for human activities.

I think it is difficult to dissect the cause of the pattern ( lower neutral diversity, higher diversity at non-synonymous sites (I just took this 0-fold/4-fold ratio as dN/dS, am I correct? No! Comments from Jeff, “0-fold and 4-fold are stricter than non and syn because not all nonsyn changes are 0-fold and not all synonymous changes are 4-fold.”, and need to figure it out later), and more deleterious variants in dogs than wolfs ) as relaxed selection and strengthened drift are concomitant.

I personally like the way they partitioned the neutral, synonymous and non-synonymous variants across the genome, and calculated the heterozygosity respectively. The figure showing the negative linear correlation of neutral heterozygosity and 0/4-fold heterozygosity is very explanatory. It looks like a good way to present the pattern.

Thinking about the idea testing the hitchhiking of deleterious variants around the selective sweeps? Will it be a good add-on to my manuscript? I did the variant effect predictor analysis, but we did not compare the mis-sense variants in the sweep and non-sweep regions. This sweep should be corresponding to highland adaptation, if we want to test this hypothesis. And that is still on the way.

In general, I like the writing of the paper. It started with a pattern, and then gradually dig into the question. It gave me some inspiration on writing my manuscript.

